packages/base/all/vendor-config-onl/src/python/onl/platform/base.py

```diff
@@ -553,3 +553,8 @@ class OnlPlatformPortConfig_20x100(object):
 class OnlPlatformPortConfig_16x10_8x25_2x100(object):
     PORT_COUNT=26
     PORT_CONFIG="16x10 + 8x25 + 2x100"
+
+class OnlPlatformPortConfig_46x10_6x100(object):
```

Comment:
@JohnsonYJLu Was adding this PortConfig intended? The __init__.py for this platform specified OnlPlatformPortConfig_32x100

Reply:
Yes, this platform use 46Ports 10G SFP and 6Ports 100G QSFP28.
Thank you for your review, I modify new init.py, and push this.